pkg/identifiers/validate_windows.go

```diff
@@ -1,41 +0,0 @@
-/*
```

Comment:
Moved inside the store.
These things are restrictions linked to the filesystem implementation, and not conceptually to restrictions we necessarily want to apply to identifiers.
Put otherwise: they are orthogonal concerns.